devices.js

```diff
@@ -13318,6 +13318,18 @@ const devices = [
         description: 'LED white',
         extend: preset.light_onoff_brightness(),
     },
+    {
+        fingerprint: [{modelID: 'TLSR82xx', manufacturerName: 'AwoX'}],
```

Comment:
Please fill in the complete fingerprint like: https://github.com/Koenkk/zigbee-herdsman-converters/pull/2368/files#

You can find this info in the data/database.db file.

Reply:
data in `database.db`:
```
{"id":8,"type":"Router","ieeeAddr":"0xa4c138ccabf1af35","nwkAddr":36305,"manufId":4417,"manufName":"AwoX","powerSource":"Mains (single phase)","modelId":"TLSR82xx","epList":[1,3],"endpoints":{"1":{"profId":260,"epId":1,"devId":258,"inClusterList":[0,3,4,5,6,8,768,4096],"outClusterList":[6,25],"clusters":{"genBasic":{"attributes":{"modelId":"TLSR82xx","manufacturerName":"AwoX","powerSource":1,"zclVersion":3,"appVersion":0,"stackVersion":2,"hwVersion":0,"swBuildId":"0122052017"}}},"binds":[],"configuredReportings":[],"meta":{}},"3":{"profId":49152,"epId":3,"devId":258,"inClusterList":[65360,65361],"outClusterList":[65360,65361],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":0,"stackVersion":2,"hwVersion":0,"swBuildId":"0122052017","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1615036764282}

```
as far as I understand we should add fingerprint like this:)? 
```
fingerprint: [
            {type: 'Router', manufacturerName: 'AwoX', modelID: 'TLSR82xx', endpoints: [
                {ID: 1, profileID: 260, deviceID: 258, inputClusters: [0, 3, 4, 5, 6, 8, 768, 4096], outputClusters: [6, 25]},
                {ID: 3, profileID: 49152, deviceID: 258, inputClusters: [65360, 65361], outputClusters: [65360, 65361]},
            ]},

```
@9mrcookie9  are you planning to extend this PR? I would love to see support for this bulbs in master :) If you don't have time I can extend this PR with this needed changes :)